ftp-upload/.nvmrc

```diff
@@ -1 +1 @@
-v10
\ No newline at end of file
+v14
```

Comment:
It seems like we would usually go for more specific version numbers to ensure consistency between environments (e.g. `14.0.0`), but since this is a lambda we can probably be more lax. Or perhaps there is specific lambda runtime behaviour I'm not aware of - either way, LGTM.